Habitica/src/com/magicmicky/habitrpgwrapper/lib/models/tasks/Task.java

```diff
@@ -112,8 +117,10 @@ public void setId(String id) {
     /**
      * @return the notes
      */
-    public String getNotes() {
-        return notes;
+    public CharSequence getNotes() {
+        CharSequence processed = Html.fromHtml(processor.markdownToHtml(EmojiParser.parseEmojis(notes)));
```

Comment:
No you can keep it open and update it as you go.
Let us know if you need any assistance!